Issue 43144: Search result hit for ontology concept should point to concept browser

Sigmonia opened the pull request:
#### Rationale
Issue 43144: Search result hit for ontology concept should point to concept browser

#### Related Pull Requests
* https://github.com/LabKey/ontology/pull/61

#### Changes
* Added OntologyBrowserPage wrapper and initialCode prop to OntologyBrowserPanel

Sigmonia committed:
Issue 43144: Search result hit for ontology concept should point to concept browser

Sigmonia committed:
Issue 43144: Search result hit for ontology concept should point to concept browser

Sigmonia committed:
Draft release notes

Sigmonia committed:
Draft release notes

cnathe commented on packages/components/src/index.ts:
```diff
@@ -1258,6 +1258,7 @@ export {
     mountWithServerContextOptions,
     // Ontology
     OntologyBrowserPanel,
+    OntologyBrowserPage,
```
I don't think that the `OntologyBrowserPanel` component is not being used anymore outside of the @labkey/components package. I believe it can be removed from this index.ts in that case.

cnathe commented on packages/components/src/internal/components/ontology/OntologyBrowserPanel.tsx:
```diff
@@ -21,6 +22,29 @@ export interface OntologyBrowserProps {
+                const loadingConcept = await fetchConceptForCode(initConceptCode);
+                setConcept(loadingConcept);
+                setLoading(false);
+            }) ();
```
We should catch the error case where the concept cannot be found. I tested this locally by editing the URL to be a concept code that didn't exist and my page just stays in the loading spinner state. As an example for how to display / handle the error case, you can look at the `OntologyBrowserPanel` where if you give it an invalid `ontologyId` param it shows the error and falls back to the display without that param.

Sigmonia committed:
CR:
 - Remove old action
 - Update nav trail
 - Switch concept to code for consistency
 - Add error handling for code not found

Sigmonia committed:
CR:
 - Remove old action
 - Update nav trail
 - Switch concept to code for consistency
 - Add error handling for code not found

cnathe commented:
@labkey-ians looks like there are some ontology module selenium test failures to look at on this branch: https://teamcity.labkey.org/viewType.html?buildTypeId=LabKey_Trunk_LabkeyPremiumTrunk_GitModules_OntologyPostgres&branch_LabKey_Trunk_LabkeyPremiumTrunk_GitModules=43144_initOntBrowser&tab=buildTypeStatusDiv

Sigmonia committed:
v2.91.0-fb-43144-initOntBrowser.2

XingY committed:
Issue 44277: SM: creating aliquots using Experiment.saveRuns api (#670)

cnathe committed:
Merge 21.11 patch fixes to develop (#672)

### version 2.91.2
*Released*: 15 November 2021
* Merge release21.11-SNAPSHOT to develop
* Includes changes from versions 2.90.2 and 2.90.3

labkey-nicka committed:
AppContext: React context for serving global application context (#673)

Sigmonia committed:
Final release notes

Sigmonia committed:
Merge remote-tracking branch 'origin/fb_43144_initOntBrowser' into fb_43144_initOntBrowser

# Conflicts:
#	packages/components/package.json
#	packages/components/releaseNotes/components.md

Sigmonia committed:
v2.93.0
